### ISSUE TITLE
Test coverage

### ISSUE THREAD

**Issue by NGMarmaduke** [scf4/callbaxx#13]
Glad to see this project hasn't succumb to the shackles of test driven development or the burden that is test coverage.

Perhaps you should add a note to the contributing guidelines to make sure contributors don't waste their time writing pointless tests. 

Real developers know when their code works.

**Comment by scf4** [comment]
Good idea. TDD doesn't even make sense. If you want to "test" something, just try it out!

And besides, that time could be much better spent on benchmarks. I've been running some preliminary benchmarks and some of these functions appear to be about 10–100x faster than the standard functions.

If my calculations are correct, Callbaxx.js is going to be a real game changer.

Checkmate, sixers :)
